### PR TITLE
[Merged by Bors] - remove unneeded msaa explicit addition from examples

### PR DESCRIPTION
**Pull request by Vrixyz**
# Objective

- Coming from https://github.com/bevyengine/bevy/pull/4797/files/7a596f1910c97c41f195b12d33025999e81dc0e5#r876310734
- Simplify the examples regarding addition of `Msaa` Resource with default value.

## Solution

- Remove addition of `Msaa` Resource with default value from examples,

### CODE REVIEW

File: examples/android/android.rs [hunk 1]
```diff
@@ -4,7 +4,6 @@ use bevy::prelude::*;
 #[bevy_main]
 fn main() {
     App::new()
-        .insert_resource(Msaa { samples: 2 })
```
[the remark this answers is not in the document]
Comment (mockersf) [reply examples/android/android.rs:7]: android is broken is several ways, I think I remember this was one of those 👍

**Review by mockersf — APPROVED** [review]

**Review by devil-ira — APPROVED** [review]

**Comment by Weibye** [comment]
Is the Msaa resource part of the default plugins?

**Comment by Vrixyz** [comment]
> Is the Msaa resource part of the default plugins?

Yes, it's added (there)[https://github.com/bevyengine/bevy/blob/fed93a0edce9d66586dc70c1207a2092694b9a7d/crates/bevy_render/src/view/mod.rs#L30] : following DefaultPlugin -> RenderPlugin -> ViewPlugin ; with its default being currently 4.

**Review by james7132 — APPROVED** [review]

**Review by Weibye — APPROVED** [review]
👍

**Pull request closed without merging**